gradle.properties

```diff
@@ -28,3 +28,132 @@ kotlin.native.ignoreDisabledTargets=true
 android.namespacedRClass=true
 
 splitties.version=3.0.0-dev-038
+
+# The plugin https://github.com/jmfayard/buildSrcVersions
+# allows to overwrite plugin and dependencies versions from the values inside 'gradle.properties'
+# Set to 'verbose' to understand how dependency and plugin versions are overwritten
+# Set to 'false' to disable this feature
+resolutionStrategyConfig=true
+
+
+# Dependencies and Plugin versions with their available updates
+# Generated by $ ./gradlew refreshVersions
+# You can edit the rest of the file, it will be kept intact
+# See https://github.com/jmfayard/buildSrcVersions/issues/77
+plugin.com.github.ben-manes.versions=0.25.0
+plugin.org.gradle.kotlin.kotlin-dsl=1.3.1
+plugin.de.fayard.buildSrcVersions=0.6.5
+plugin.com.gradle.plugin-publish=0.10.1
+plugin.com.gradle.build-scan=2.4.2
+version.androidx.localbroadcastmanager..localbroadcastmanager=1.0.0
+version.org.jetbrains.kotlinx..kotlinx-coroutines-core-native=1.3.2
+version.androidx.asynclayoutinflater..asynclayoutinflater=1.0.0
```

Comment:
Could `buildSrcVersions` sort the versions lines before writing them to `gradle.properties`?
Would make eye scanning much easier.

Reply:
It's possible that sorting would have to be performed early on, or it could be a little tricky as the shown version updates still need to be next to their artifacts, and not sorted all at the top or the bottom.

Reply:
BTW, it looks like there's currently a sort strategy based on the length of the property key, from longest to shortest.